src/containers/Sidebar.js

```diff
@@ -8,23 +8,37 @@ import Splitter from '../components/Splitter';
 import { fetchCollections } from '../ducks/collections';
 import { capitalize } from '../utils/helpers';
 import { sidebar as SidebarTranslations } from '../translations';
+import classnames from 'classnames';
 import _ from 'underscore';
 
 export class Sidebar extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
```

Comment:
You don't need this. You can use arrow function for class methods.

Reply:
I used this syntax so there'd be one less offense while migrating to webpack via #502

Reply:
Yeah, we will definitely fix that offence but not by changing our codebase. I'm gonna look into it.